lib/robots/dor_repo/assembly/content_metadata_create.rb

```diff
@@ -18,7 +18,7 @@ def perform_work
           return LyberCore::ReturnState.new(status: :skipped, note: 'No stubContentMetadata to load from the filesystem') unless assembly_item.stub_content_metadata_exists?
 
           tries = 0
-          max_tries = 3
+          max_tries = 0
```

Comment:
yes, i know, this makes it look like it won't do anything at all, but if you examine the slightly wonky logic, it'll do it exactly once and then raise if there an error